entries/g/grab.com.json

```
@@ -0,0 +1,22 @@
+{
+  "Grab": {
+    "passwordless": "allowed",
+    "documentation": "https://engineering.grab.com/embracing-passwordless-authentication-with-passkey",
+    "regions": [
+      "sg",
+      "my",
+      "kh",
+      "id",
+      "mm",
+      "ph",
+      "vn",
+      "th"
+    ],
+    "categories": [
+      "food",
+      "payments",
+      "retail",
+      "transport"
```

```suggestion
      "transport",
      "postal",
      "insurance",
      "travel",
      "banking"
```

> **Express**
> Send packages, documents, and beyond.

> **Insurance**
> Get everyday protection with accessible insurance.

> **Hotels**
> Discover and book amazing hotel stays.

They appear to have a banking service as well, given that [their help centre](https://help.grab.com/passenger/en-sg/6567872428569) mentions a "GXS/GX Bank account".

---

Yes - I wasn't sure how many categories to put it in because as a super-app they have their fingers in a lot of pies!
Regarding travel - I didn't see that on the [list of categories](https://github.com/2factorauth/passkeys/blob/master/CONTRIBUTING.md#categories). Is it a new addition?

---

> Yes - I wasn't sure how many categories to put it in because as a super-app they have their fingers in a lot of pies!

I tend to put as many as the service has

> Regarding travel - I didn't see that on the [list of categories](https://github.com/2factorauth/passkeys/blob/master/CONTRIBUTING.md?#categories). Is it a new addition?

The categories in the Passkeys Directory's CONTRIBUTING.md are out-of-sync. The categories are synced with [2FA Directory](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#categories) though. I'll update the list here when I get a chance.